packages/dotcom-build-image-uploader/README.md

```diff
@@ -0,0 +1,62 @@
+# @financial-times/dotcom-build-image-uploader
```

Comment:
I think this package name doesn't seem right because it doesn't actually upload anything. Perhaps `dotcom-build-images` would be more appropriate?

Reply:
good shout! i think i was getting carried away with the jira ticket description